src/main/java/at/hannibal2/skyhanni/features/garden/AtmosphericFilterDisplay.kt

```diff
@@ -0,0 +1,71 @@
+package at.hannibal2.skyhanni.features.garden
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.GuiRenderEvent
+import at.hannibal2.skyhanni.events.LorenzTickEvent
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.RenderUtils.renderString
+import at.hannibal2.skyhanni.utils.StringUtils.matchMatcher
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import io.github.moulberry.notenoughupdates.util.SkyBlockTime
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class AtmosphericFilterDisplay {
+
+    private val config get() = SkyHanniMod.feature.garden.atmosphericFilterDisplay
+
+    private val atmosphericFilterDisplayGroup = RepoPattern.group("atmosphericfilter.hud")
```

Comment:
group name should start with "garden" also group name doesnt have to be this long imo "patternGroup" would be fine as there are no other groups in this file